ledger-tool/src/main.rs

```diff
@@ -3449,7 +3449,7 @@ fn main() {
                                 eprintln!("Unable to create incremental snapshot without a base full snapshot");
                                 exit(1);
                             }
-                            let full_snapshot_slot = starting_snapshot_hashes.unwrap().full.hash.0;
+                            let full_snapshot_slot = starting_snapshot_hashes.unwrap().full.0 .0;
```

Comment:
I think GitHub is just rendering the diff weird for me. I'm seeing `full.0.0;` in a bunch of places, but I don't think this would compile if that was actually the case.

Reply:
Ah, gross. Would it be possible to add an accessor function to get the slot?